index.js

```diff
@@ -94,9 +98,46 @@ exports.scrapeCOinS = function(html, callback){
 	callback(null);
 };
 
-// TODO
+/**
+ * Scrapes Dublin Core data given html object
+ * @param  {Object}   html     html Cheerio object
+ * @param  {Function} callback callback(dublinCoreDataObject)
+ */
 exports.scrapeDublinCore = function(html, callback){
-	callback(null);
+	var meta = {},
+		metaTags = $('meta,link');
+
+	metaTags.each(function() {
+		var element = $(this),
+			isLink = this.name === 'link',
+			nameAttr = element.attr(isLink ? 'rel' : 'name');
+
+		// If the element isn't a Dublin Core property, skip it
+		if (!nameAttr
+			|| (nameAttr.substring(0, 3) !== 'DC.'
```

Comment:
dc can be lowercase, see: 

http://dublincore.org/documents/dcq-html/ section 3.